pandas/core/indexes/multi.py

```diff
@@ -379,14 +379,21 @@ def _validate_codes(self, level: list, code: list):
             code = np.where(null_mask[code], -1, code)  # type: ignore[assignment]
         return code
 
-    def _verify_integrity(self, codes: list | None = None, levels: list | None = None):
+    def _verify_integrity(
+        self,
+        codes: list | None = None,
+        levels: list | None = None,
+        levels_to_verify: list | None = None,
```

Comment:
`list[int] | None` is more precise and it is clearer that we've not supplying level names.

Reply:
Done